test/maps/lua_testsuite.wmf/scripting/gettext.lua

```diff
@@ -6,9 +6,9 @@ test_descr = lunit.TestCase("Gettext Test")
 -- Code takes a different path when textdomain was pushed, so we have 2 variants of each test.
 
 function gettext_calls()
-   assert_equal("foo", _"foo")
    assert_equal("foo", _("foo"))
-   assert_equal(_"Carrier", _("Carrier"))
+   assert_equal("foo", _("foo"))
```

Comment:
here we should still ensure the case without brackets would work. so this shpuld be reverted.